### DISCUSSION TITLE
Adding telescope-live-grep-args?

### DISCUSSION THREAD

**Discussion by sp71** [LazyVim/LazyVim#2292] [Q&A]
Trying to add [telescope-live-grep-args](https://github.com/nvim-telescope/telescope-live-grep-args.nvim) but have no luck in getting it to work. I've posted my [config](https://github.com/sp71/config/blob/main/lua/plugins/example.lua#L71) online if anyone has ideas.

**Comment by dpetka2001** [comment]
Take a look at #1918. It's the same thing.

**Reply by dpetka2001** [reply]
Also don't put your configuration in the `example.lua`. That is just there for reference and won't actually load any of your configuration. Create another file in `/lua/plugins/` directory and put your configuration there.

**Comment by ejsadiarin** [comment]
try this @sp71 (in my `lua/plugins/telescope.lua`):
- i replaced my `<leader>sg` to use live_grep_args (instead of live_grep)
- i also added `<leader>sG` to grep from my home directory

```lua
return {
    "nvim-telescope/telescope.nvim",
    -- existing configs --
    keys = {
      {
        "<leader>sg",
        function()
          require("telescope").extensions.live_grep_args.live_grep_args({
            -- cwd = require("telescope.utils").buffer_dir(),
          })
        end,
        desc = "Live Grep (root/dynamic)",
      },
      {
        "<leader>sG",
        function()
          local dir = vim.env.HOME .. "/"
          require("telescope").extensions.live_grep_args.live_grep_args({
            search_dirs = { dir },
            additional_args = { "--hidden" },
          })
        end,
        desc = "Live Grep from Home",
      },
      {
        "<leader>fH",
        function()
          local dir = vim.env.HOME .. "/"
          require("telescope.builtin").find_files({
            find_command = { "fd", "-tf", "--hidden", "-E", { "node_modules", ".git", ".cargo" }, "--search-path", dir },
            prompt_prefix = "~/ ",
          })
        end,
        desc = "Find files from Home",
      }
      -- other keymaps... --
    }
}
```
